tests/test_output_formats.py

```diff
@@ -771,53 +773,61 @@ def test_security_data_output_format_has_expected_options():
 
 
 class TestDataFrameOutputFormatter:
-    def test_init_sets_format_func_to_formatted_json_function_when_json_format_option_is_passed(
-        self, mock_dataframe_to_json
-    ):
-        output_format = output_formats_module.OutputFormat.RAW
-        formatter = output_formats_module.DataFrameOutputFormatter(output_format)
-        formatter.echo_formatted_dataframe(TEST_DATAFRAME)
-        mock_dataframe_to_json.assert_called_once_with(
-            TEST_DATAFRAME,
-            orient="records",
-            lines=False,
-            index=True,
-            default_handler=str,
+    test_df = DataFrame(
+        [
+            {"string_column": "string1", "int_column": 42, "null_column": None},
+            {"string_column": "string2", "int_column": 43, "null_column": NaN},
+        ]
+    )
+
+    def test_format_when_none_passed_defaults_to_table(self):
+        formatter = DataFrameOutputFormatter(output_format=None)
+        assert formatter.output_format == OutputFormat.TABLE
+
+    def test_format_when_unknown_format_raises_cli_exception(self):
+        with pytest.raises(Exception):
```

Comment:
when we use a bare exception, anything could have gone wrong here, not necessarily the thing that we're testing went wrong. A more specific type of exception (doesn't necessarily have to be a custom exception) would improve this